tests/settings.py

```diff
@@ -21,14 +21,16 @@
     # local sqlite database file
     DATABASES['default'] = {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': INSTALL_DIR + 'db.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
Just to be clear, the problem is the INSTALL_DIR in tests/settings.py (not the one in the Makefile):
tests/settings.py:INSTALL_DIR = '/usr/local/altschool/dynamic-rest/'